addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -830,10 +848,16 @@ MockServer.include({
             if (!lastMessage) {
                 continue;
             }
+            // deprecated: update mail.channel/fetched_message_id
             this._mockWrite('mail.channel', [
                 [channel.id],
                 { fetched_message_id: lastMessage.id },
             ]);
+            const [cp] = this._getRecords('mail.channel.partner', [['channel_id', 'in', [channel.id]], ['partner_id', '=', this.currentPartnerId]]);
```

Comment:
mix of operator "in" and "=", both are correct but lack of consistency is strange